benchmark/client/App.vue

```diff
@@ -9,50 +16,47 @@ const selected = ref<number>()
 const rows = shallowRef<
   {
     id: number
-    label: string
+    label: ShallowRef<string>
   }[]
 >([])
 
-function setRows(update = rows.value.slice()) {
-  rows.value = update
-}
-
+// Bench Add: https://jsbench.me/45lzxprzmu/1
 const add = wrap('add', () => {
-  rows.value = rows.value.concat(buildData(1000))
+  rows.value.push(...buildData(1000))
+  triggerRef(rows)
```

Comment:
I tested different options for [adding items to an existing arrow on jsbench.me](https://jsbench.me/45lzxprzmu/1). this is the fastest.